docs/dev.rst

```diff
@@ -1,20 +1,522 @@
+
+
+
+Set Up a Dev Org
 ----------------
 
-Track and retrieve changes
---------------------------
+The ``dev_org`` flow creates an org to develop on by moving all metadata (managed and unmanaged) into the org, and configuring it to be ready for development.
+
+.. note:: Run ``cci flow info dev_org`` for a full list of the ``dev_org`` flow steps.
+
+To run the ``dev_org`` flow against the project's `default org<TODO>`_:
+
+.. code-block:: console
+
+    $ cci flow run dev_org
+
+You can also use the ``--org`` option to explicitly list the org.
+
+    Example: Run the ``dev_org`` flow against the org currently defined as ``dev`` in CumulusCI.
+
+.. code-block:: console
+
+    $ cci flow run dev_org --org dev
+
+..
+
+    Open the new ``dev`` org to begin development.
+
+.. code-block:: console
+
+    $ cci org browser dev
+
+
+Set the Capture State
+^^^^^^^^^^^^^^^^^^^^^
+
+To make changes to capture in an org, use the ``snapshot_changes`` task to create a snapshot. A snapshot tells the Salesforce CLI source tracking to set the org's current state as a baseline for changes to be made against.
+
+.. code-block:: console
+
+    $ cci task run snapshot_changes --org dev
+
+.. note:: There's no need to run ``snapshot_changes`` if you've set up a scratch org with any of these flows:
+
+    * ``dev_org``
+    * ``dev_org_namespaced``
+    * ``qa_org``
+    * ``regression_org``
+    * ``install_beta``
+    * ``install_prod``
+    
+    These flows all run ``snapshot_changes`` as their last step.
+
+The ``list_changes`` and ``retrieve_changes`` tasks detect new metadata but ignore any prior changes.
+
+To check that the snapshot was created successfully:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev
+
+
+
+List Changes
+------------
+
+To see what components have changed in a target org:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev
+
+.. note::
+    
+    This functionality relies on Salesforce's source tracking feature, so it's only available in scratch orgs.
+
+To include/exclude components from the list using the ``--include`` or ``--exclude`` option:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev --include "test.*,another_regex" --exclude "something_to_exclude"
+
+The ``include`` and ``exclude`` patterns are matched against both the metadata type and name of the component.
+
+To include all changed components of specific types:
+
+.. code-block:: console
+
+    $ cci task run list_changes --org dev --types "CustomObject,CustomField"
+
+
+
+Retrieve Changes
+----------------
+
+The ``retrieve_changes`` task supports both ``sfdx`` and ``mdapi`` formatted source code. It utilizes the `SourceMember <https://developer.salesforce.com/docs/atlas.en-us.api_tooling.meta/api_tooling/tooling_api_objects_sourcemember.htm>`_ sObject to detect what has changed in an org, but also lets you be more selective regarding which components to retrieve when compared to the ``dx_pull`` task.
+
```

Comment:
Will most advanced devs know what the dx_pull task is?

Reply:
We could put a note here, that says something like: "The `dx_pull` task is a wrapper for the `sfdx force:source:pull` command. This command pulls _all_ changed source from the scratch org into your project's repository."